test/integration/helpers.js

```diff
@@ -212,7 +213,8 @@ function _spawnMochaWithListeners(args, fn, opts) {
   mocha.on('close', function(code) {
     fn(null, {
       output: output.split('\n').join('\n'),
```

Comment:
Isn't this a noop?

Reply:
yes, I think so. that is odd indeed.

Reply:
please make an issue to explore if this can safely be removed